### DISCUSSION TITLE
Creation of detailed service log

### DISCUSSION THREAD

**Discussion by SMetriq** [Budibase/budibase#4455] [Help]
Hi,

I followed steps on budibase.com and I have now two table (vehicles and service log). I would like to create a page where you can select a service log id and all information are showed in different textbox.

How is it possible to do this ? 

Thanks

**Comment by mjashanks** [comment]
Hi @SMetriq 

So, it's probably best to have a look at the auto-generated edit screens to understand how this works.

Firstly, they have a URL that looks like `/table/:id`. The `:id` part is a URL parameter, which you can subsequently use on your edit screen via the binding `{{ URL.id }}`. You can name this parameter whatever you like.

On the edit screen, there's is a `Data Provider`, setup like...
- Data - points to your table
- Limit 1 - i.e. only returns the first row that it finds
- in `Define Filters`, it does something like `_id equals {{ URL.id }}` - i.e. returning the record that matched the `:id` URL parameter.

Inside the `Data Provider`, is a Repeater. This will "Repeat" for each row returned b the data provider (there should only be one in this instance)

Now, you can put whatever you need inside this repeater, depending on what you want to display. You will be able to bind to any of the fields in the row that has been returned.
